src/modules/logger/logged_topics.cpp

```diff
@@ -109,6 +109,7 @@ void LoggedTopics::add_default_topics()
 	add_topic("vehicle_status_flags");
 	add_topic("vtol_vehicle_status", 200);
 	add_topic("yaw_estimator_status", 200);
+	add_topic("optical_flow_vel", 200);
```

Comment:
Generally I try to keep anything like this sorted alphabetically. Lists of topics, headers included, etc.